docs/source/install/upgrades.rst

```diff
@@ -167,6 +167,38 @@ The following sections call out the migration scripts that need to be run when u
 respective version. If you are upgrading across multiple versions, make sure you run the scripts for
 any skipped versions:
 
+v3.4
+''''
+
+*  |st2| now uses python 3 on RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3.
+
+
+* If SELINUX is enabled, install the python 3 SELinux rules with:
+
+  .. sourcecode:: bash
+
+    sudo yum install -y libselinux-python3
+
+* Ensure python3-devel can be installed from an enabled repository:
```

Comment:
Is the `python3-devel` absence only an issue for RHEL systems?
If so, worth mentioning that so CentOS users don't need to worry about this instruction.